tutorials/Multi-Agent Reinforcement Learning Training with WarpDrive.ipynb

```diff
@@ -0,0 +1,313 @@
+    "\n",
+    "We have integrated WarpDrive with the [Pytorch Lightning](https://www.pytorchlightning.ai/) framework, which greatly reduces the trainer boilerplate code, and improves training flexibility.\n",
+    "\n",
+    "Below, we demonstrate how to use WarpDrive and PytorchLightning together to train a game of [Tag](https://github.com/salesforce/warp-drive/blob/master/example_envs/tag_continuous/tag_continuous.py) where multiple *tagger* agents are trying to run after and tag multiple other *runner* agents. As such, the Warpdrive framework comprises several utility functions that help easily implement any (OpenAI-)*gym-style* RL environment, and furthermore, priovides quality-of-life tools to train it end-to-end using just a few lines of code. You may familiarize yourself with WarpDrive with the help of these [tutorials](https://github.com/salesforce/warp-drive/tree/master/tutorials).\n",
+    "\n",
+    "We invite everyone to **contribute to WarpDrive**, including adding new [multi-agent environments](https://github.com/salesforce/warp-drive/tree/master/example_envs), proposing new features and reporting issues on our open source [repository](https://github.com/salesforce/warp-drive)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9f37d334",
+   "metadata": {},
+   "source": [
+    "### Dependencies: this notebook requires the rl-warp-drive as well as the pytorch-lightning packages."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e77cc5cb",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "! pip install --quiet rl_warp_drive pytorch_lightning"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "50124771",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import argparse\n",
+    "import numpy as np\n",
+    "import torch\n",
+    "\n",
+    "from example_envs.tag_continuous.tag_continuous import TagContinuous\n",
+    "from warp_drive.env_wrapper import EnvWrapper\n",
+    "from warp_drive.training.lightning_trainer import WarpDriveModel, PerfStatsCallback\n",
+    "from warp_drive.training.utils.data_loader import create_and_push_data_placeholders\n",
+    "\n",
+    "from pytorch_lightning import LightningModule, Trainer, seed_everything\n",
+    "from pytorch_lightning.callbacks import ModelCheckpoint"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "be8ad0a4",
+   "metadata": {},
+   "source": [
+    "# Specify a set of run configurations for your experiments"
```

Comment:
Add cell above with a short description + image of Tag + link to the whitepaper + WarpDrive repo for more background